internal/jobservice/eventstojobs/event_job_response.go

```diff
@@ -0,0 +1,36 @@
+package eventstojobs
+
+import (
+	"fmt"
+
+	"github.com/G-Research/armada/pkg/api"
+	"github.com/G-Research/armada/pkg/api/jobservice"
+)
+
+func EventsToJobResponse(message api.EventMessage) (*jobservice.JobServiceResponse, error) {
```

Comment:
Why do we need this? Are not just interested in tracking the current state of each job?

Reply:
In short, the job response only cares about terminal events and running.  So we only return those but there are 14-15 other events that can happen so we use this to translate events into our jobresponse.